tests/test_display_options_integration.py

```
@@ -155,9 +169,20 @@ async def render_display_option(
+            "zone, place",
+            "not_home, Roy Spiegel MSW, house, Koreatown, 1, Bridge Plaza North",
+        ),
+        (
+            "zone_name, place",
+            "not_home, Roy Spiegel MSW, house, Koreatown, 1, Bridge Plaza North",
+        ),
         ("formatted_place", "Roy Spiegel MSW, Fort Lee, NJ"),
+        (
+            "osm_formatted_address",
+            "Roy Spiegel MSW, 1, Bridge Plaza North, Koreatown, Fort Lee, Bergen County, "
+            "New Jersey, 07024, United States",
```

Ruff (0.16.1)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)